core/src/auctions.ts

```diff
@@ -110,6 +111,27 @@ export const fetchAllInitialAuctions = async function (network: string): Promise
     return auctionGroups.flat();
 };
 
+export const fetchAllAuctionsFromCollateralTypes = async function (
+    network: string,
+    collateralTypes: string[] | undefined
+): Promise<AuctionInitialInfo[]> {
```

Comment:
The title is misleading, I would instead just extended existing `fetchAllInitialAuctions` with optional `collateralTypes` prop